user/lxqt-panel/template.py

```diff
@@ -23,6 +23,7 @@
     "lm-sensors-devel",
     "lxqt-globalkeys-devel",
     "lxqt-menu-data",
+    "qt6-qtbase-private-devel",
```

Comment:
```suggestion
    "qt6-qtbase-private-devel",  # qwaylandscreen_p.h
```
the specific header used

Reply:
actually not really required ig